src/styled-components/reusables.js

```diff
@@ -0,0 +1,155 @@
+import styled from "styled-components";
```

Comment:
styled component are great and I like them a lot! They make it so much easier (to me) to locate the relevant CSS. 

With that being said, what do you think about putting the CSS into the component files itself? Eg. the code between lines 3-20 below right into the `MainCont` file?

Reply:
i use MainCont for more than just one thing so I didn't want it to have fixed design features such as background etc.. I could create a new styled-component that is just for the whole page.

Reply:
Sorry I'm not sure what you mean ? there isn't a MainCont file